storage/innobase/xtrabackup/doc/source/version-check.rst

```diff
@@ -4,63 +4,113 @@
 Version Checking
 ================================================================================
 
-Complex software projects (such as MySQL flavors) always have versions with
-minor or major compatibility problems, regressions and bugs, etc. It is too
-difficult to take care about all these compatibility nuances for a human, and
-this is the reason for a |version-check| feature to exist.
+Some Percona software contains “version checking” functionality which is a
+feature that enables Percona software users to be notified of available software
+updates to improve your environment security and performance. Alongside this,
+the version check functionality also provides Percona with information relating
+to which software versions you are running, coupled with the environment
+confirmation which the software is running within. This helps enable Percona to
+focus our development effort accordingly based on trends within our customer
+community.
 
-The feature allows compatible tools to query the Version Check database and
-print advice on any potential risks associated with any of the relevant
-software versions.
+
+The purpose of this document is to articulate the information that is
+collected, as well as to provide guidance on how to disable this functionality
+if desired.
 
 Usage
------
+=====
 
 *Version Check* was implemented in |pt| 2.1.4, and was enabled by default in
-version 2.2.1. Currently it is supported by most tools in |pt|, |pxb|, and
-|pmm|. In |Percona XtraBackup|, this feature was introduced in version 2.1.5. By
-default, it is also enabled.
+version 2.2.1. Currently it is supported as a ``--[no]version-check`` option
```

Comment:
Speaking of xtrabackup, version check has been added in 2.1.5 and was always enabled by default.

Reply:
I think multi-purpose toolkit/xtrabackup version of this document is ok for now.